idd/Energy+.idd.in

```diff
@@ -24823,6 +24823,55 @@ AirflowNetwork:SimulationControl,
       \key Yes
       \key No
       \default No
+ A10, \field Duct Sizing Method
```

Comment:
I would increase `\min-fields` to 16 so it includes this field.

Reply:
I am going to make changes. Hope the diffs go awy.

I tested tbl.htm files from Window test and no differences were found.

Reply:
@mjwitte I upload a zip file, containing two files of tabl.htm from develop and my branch. I checked difference using WidDiff. The differences are time stamps.
[AirflowNetwork3zVent_Table.zip](https://github.com/NREL/EnergyPlus/files/8846824/AirflowNetwork3zVent_Table.zip)